.gitignore

```diff
@@ -1 +1,7 @@
 /.idea
+/bin/*
+!/bin/seq1.fasta
+!/bin/seq2.fasta
+!/bin/out-seq1.cfrk
+!/bin/out-seq2.cfrk
+!/bin/out.cfrk
```

Comment:
Qual a diferença deste para os demais arquivos de saída (`out-seq1.cfrk` e `out-seq2.cfrk`) ? Se ele é diferente, não deveria ter um arquivo de entrada para ele? Se ele for igual, não deveria ficar fora do repositório?

Reply:
Bom, os nomes dos arquivos ficaram confusos mesmo. O out-seq1.cfrk é o resultado da execução do cfrk original para seq1.fasta e o out-seq2.cfrk é o resultado da execução do cfrk original para seq2.fasta. A ideia é que o Travis execute o build dele para gerar o out.cfrk e fazer as comparações com out-seq1.cfrk e out-seq2.cfrk. Como o Travis não consegue rodar o cfrk com CUDA eu tive que manter o out.cfrk pra validar o teste...

Tem alguma sugestão de como podemos melhorar esse processo?

Reply:
Bem, pelo que entendi, o CI não está validando nada, né?
Sendo assim, não faz sentido esse arquivo `out.cfrk`, bem como fazer o CI falhar - pois "ainda" não há nada a testar.
Quando o código em CPU estiver funcionando, precisaremos mudar o CI para que ele realmente faça o teste com os 2 arquivos de entrada, comparando-os com os arquivos de saída.

Reply:
Concordo! Vou retirar o out.cfrk e deixar somente o build no script do CI por enquanto. Pelo menos temos o ambiente de teste encaminhado agora. rs